client/coral-framework/components/Slot.js

```diff
@@ -1,16 +1,22 @@
 import React from 'react';
 import cn from 'classnames';
 import styles from './Slot.css';
+import {connect} from 'react-redux';
 import {getSlotElements} from 'coral-framework/helpers/plugins';
 
-export default function Slot ({fill, inline = false, ...rest}) {
+function Slot ({fill, inline = false, plugin_config: {debug, ...config}, ...rest}) {
```

Comment:
A little thought: Any particular reasons why you hide the `debug` from the plugins? We could just passed it to the slots and whether they want to use it or not is up to the plugins. I think this would be more straight forward. Like "everything we passed into `plugin_config` is available to plugins" instead of "everything we passed into `plugin_config` is available to plugins except `debug`".